de.metas.adempiere.adempiere/base/src/main/java/de/metas/bpartner/service/IBPartnerBL.java

```diff
@@ -206,6 +206,10 @@
 		@Default
 		@NonNull
 		Comparator<User> comparator = Comparator.comparing(User::getName);
+
+		boolean forceActive;
```

Comment:
instead of `forceActive` i would call it `onlyActive`

Reply:
done